operator/templates/manager/manager.yaml

```diff
@@ -75,14 +75,14 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: "{{ .Release.Name }}-minio"
-                  key: accesskey
+                  key: root-user
```

Comment:
This will also require tweaks to the installation docs: https://www.securecodebox.io/docs/getting-started/installation#accessing-the-included-minio-instance